projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.html

```diff
@@ -1,22 +1,27 @@
 <div class="document-filter-container">
   <div class="form-group document-filter" [formGroup]="documentFilterFormGroup">
-    <input class="form-control document-search"
-      type="search"
-      id="document-search"
+    <input 
+      class="form-control document-search" 
+      type="search" 
+      id="document-search" 
       name="documentSearchFormControl"
-      formControlName="documentSearchFormControl"
+      formControlName="documentSearchFormControl" 
```

Comment:
You will probably find that lint complains about trailing spaces, so you should probably fix this